app/components/Notifications/Notifications.jsx

```diff
@@ -1,16 +1,30 @@
 import { useActionData } from '@remix-run/react';
 import React, { useEffect } from 'react';
 import { ToastContainer, toast } from 'react-toastify';
-import { DEFAULT_TOAST_CONFIG } from 'app/utils/constants';
+import { DEFAULT_TOAST_CONFIG, QUESTION_CREATED_TOAST_CONFIG } from 'app/utils/constants';
 import useGlobalSuccessMessage from 'app/utils/hooks/useGlobalSuccessMessage';
 
 function Notifications() {
   const globalSuccess = useGlobalSuccessMessage();
   const data = useActionData();
 
+  const successAnonMessage = (message, questionUrl) => (
```

Comment:
Notification keeps poping up after moving to different routes on wizeq is this an expected behavior?

Reply:
It is an odd behaviour of this line, happened before this PR too. I think it is saved for a short period of time so it has this behaviour sometimes for a short period of time? We can investigate this as a new ticket or I can try to look into it in here. What do you think?

Reply:
I think we can open a new ticket for this behavior, but it is up to you  😃

Reply:
I will create this new ticket and we can work on it, if it becomes too frequent/intrusive we can prioritize it 👍